app/views/requests/request_mailer/_item_info_all.html.erb

```diff
@@ -0,0 +1,27 @@
+<div style="margin-bottom:18px;font-weight:normal;font-size:16px;">Items Requested</div>
+<table style="width:100%;">
+  <% @submission.items.each_with_index do |item, index| %>
+  <tr>
+    <% unless item[:selected].empty? %>
```

Comment:
Items in a submission should only be the selected items, so this check is not needed. https://github.com/pulibrary/requests/blob/master/app/models/requests/submission.rb#L15